src/Table/Column/Filter/Model.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\ui\Table\Column\Filter;
+
+use atk4\core\NameTrait;
+use atk4\core\SessionTrait;
+use atk4\data\Field;
+use atk4\data\Persistence;
+
+/**
+ * Implement a generic Type model for filtering data.
+ */
+class Model extends \atk4\data\Model
```

Comment:
This must be changed. Model is used a lot and any confusion with it is highly unwanted. Previously, this was named "FilterModel\Generic"